src/chat/chat_line.rs

```diff
@@ -110,7 +110,7 @@ live_design! {
     ChatLineBody = <View> {
         width: Fill,
         height: Fit,
-        spacing: 12,
+        spacing: 20,
```

Comment:
This is an extra adjustment. Without the bubble, 12 as spacing between the assistant name and the message may look to tight, so I added a bit more.